.github/workflows/ci.yml

```diff
@@ -55,24 +55,29 @@ jobs:
         run: |
           mvn ${{env.MAVEN_PARMS}} compile
 
+      - name: Perform CodeQL Analysis
+        uses: github/codeql-action/analyze@v3
+
       - name: Unit Tests
         run: |
           mvn ${{env.MAVEN_PARMS}} test
 
       - name: Package
         run: |
-          mvn ${{env.MAVEN_PARMS}} package
+          mvn ${{env.MAVEN_PARMS}} package -Dmaven.test.skip=true
```

Comment:
This is preventing double runs of tests, yes?

Reply:
Yes. That is exactly what it does.